docs/devops.md

```diff
@@ -186,10 +186,10 @@ Currently a public beta testing version is available at:
 | :---------- | :------- | :--------------------------------------- |
 | Learn       | English  | <https://www.freecodecamp.dev>           |
 |             | Espanol  | <https://www.freecodecamp.dev/espanol>   |
-|             | Chinese  | <https://chinese.freecodecamp.dev>       |
+|             | Chinese  | <https://freecodecamp.dev/chinese>       |
 | News        | English  | <https://www.freecodecamp.dev/news>      |
 | Forum       | English  | <https://forum.freecodecamp.dev>         |
-|             | Chinese  | <https://chinese.freecodecamp.dev/forum> |
+|             | Chinese  | <https://freecodecamp.dev/chinese/forum> |
```

Comment:
This one I'm not really sure about. It doesn't seem like we have the English dev forum anymore.

It might be fine to remove this line entirely, but we should probably check with Mrugesh first.